analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Tests/IssueLocationCollectorTest.GetPreciseIssueLocations.cs

```diff
@@ -19,18 +19,14 @@
  */
 
 using System;
-using System.Collections.Generic;
 using System.Linq;
 using FluentAssertions;
-using Microsoft.CodeAnalysis.Text;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-namespace SonarAnalyzer.UnitTest.TestFramework.IssueLocationCollectorTests
+namespace SonarAnalyzer.UnitTest.TestFramework.Tests
 {
-    [TestClass]
-    public class IssueLocationCollector_GetPreciseIssueLocations
+    public partial class IssueLocationCollectorTest
     {
-        [TestMethod]
```

Comment:
Did you remove this by mistake? If I put it back, the test is passing.

Reply:
oops :)